src/main/java/com/box/sdk/BoxSignRequestSigner.java

```diff
@@ -0,0 +1,655 @@
+package com.box.sdk;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.box.sdk.internal.utils.JsonUtils;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * Represents a signer in BoxSignRequest.
+ */
+public class BoxSignRequestSigner extends BoxJSONObject {
```

Comment:
Can you please check the verbiage for these parameter comments against https://staging.developer.box.com/reference/resources/sign-requests/?